### DISCUSSION TITLE
Lacking of 60k data

### DISCUSSION THREAD

**Discussion by SkyYap** [enhuiz/vall-e#17] [General]
I have read through the [paper](https://arxiv.org/abs/2301.02111) and found out they are using 60k unlabeled audio data to train the AI. 

Anyone has any direction we can reproduce the method in the paper that using speech recognition to generate the transcription and increase the accuracy of it?

**Comment by eonglints** [comment]
Hey, just to say that [the Multilingual LibriSpeech dataset](https://www.openslr.org/94/) would do the trick. It's also from LibriVox and has 45k hours of transcribed speech.

**Reply by SkyYap** [reply]
> Hey, just to say that [the Multilingual LibriSpeech dataset](https://www.openslr.org/94/) would do the trick. It's also from LibriVox and has 45k hours of transcribed speech.

thanks for your sharing! Appreciate it.
